<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Container{
background-color:rgb(222, 221, 218)

}#contentArea{

	background-color:rgb(222, 221, 218)

}

QCommandLinkButton{
color:white;
}

#sideBar{
	background-color:rgb(53, 132, 228);
	color:white;
	border-radius:20px;
	

}

QFrame{
	background-color:white;
	border:1px solid rgb(192, 191, 188);border-radius:20px;
}

#dashboard{
background-color: rgb(246, 245, 244)
}#inventory{background-color:rgb(246, 245, 244);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Container" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="sideBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="sideBarDrawer" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:white;
border-radius:25px;</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="menuButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="mouseTracking">
                   <bool>true</bool>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="toolTip">
                   <string comment="Open Menus" extracomment="Open Menus"/>
                  </property>
                  <property name="statusTip">
                   <string comment="Open Menus" extracomment="Open Menus"/>
                  </property>
                  <property name="whatsThis">
                   <string comment="Open Menus" extracomment="Open Menus"/>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resource.qrc">
                    <normaloff>:/icons/menu.png</normaloff>:/icons/menu.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>50</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>true</bool>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="DashborderBtn">
               <property name="text">
                <string>Dashboard</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_9">
               <property name="text">
                <string>ADD Product To Stock</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="InventoryBtn">
               <property name="text">
                <string>InVentory</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_11">
               <property name="text">
                <string>Quick Search</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_10">
               <property name="text">
                <string>Report</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_8">
               <property name="text">
                <string>CommandLinkButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_7">
               <property name="text">
                <string>Bin / Drawer Managment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_5">
               <property name="text">
                <string>CommandLinkButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="commandLinkButton_4">
               <property name="text">
                <string>CommandLinkButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCommandLinkButton" name="settingBtn">
               <property name="text">
                <string>Settings and Preference</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="contentArea">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="dashboard">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="section" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QWidget" name="widget_15" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_28">
                       <item>
                        <widget class="QFrame" name="card_14">
                         <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QWidget" name="widget_16" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_29">
                       <item>
                        <widget class="QFrame" name="card_15">
                         <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QWidget" name="widget_14" native="true">
                      <layout class="QHBoxLayout" name="horizontalLayout_27">
                       <item>
                        <widget class="QFrame" name="card_13">
                         <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="section_2" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QWidget" name="contentContainer" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QFrame" name="maincard">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <item>
                        <widget class="QCheckBox" name="checkBox">
                         <property name="text">
                          <string>CheckBox</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="section_3" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QFrame" name="card_5">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="inventory">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
            <widget class="QWidget" name="settings">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item alignment="Qt::AlignTop">
               <widget class="QTabWidget" name="tabWidget">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="UserPreference">
                 <attribute name="title">
                  <string>Tab 1</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_7"/>
                </widget>
                <widget class="QWidget" name="DbSetting">
                 <attribute name="title">
                  <string>Tab 2</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QWidget" name="widget_12" native="true">
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Database Settings</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_8">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="checkBox_2">
                       <property name="text">
                        <string>Host Private Db</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_9">
                       <property name="text">
                        <string>Database Url</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>382</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab">
                 <attribute name="title">
                  <string>Page</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QWidget" name="widget_11" native="true">
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>preferences</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>487</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_10" native="true"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_2">
                 <attribute name="title">
                  <string>Page</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_14">
                  <item>
                   <widget class="QWidget" name="widget_30" native="true">
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <item>
                      <widget class="QLabel" name="about">
                       <property name="text">
                        <string>about</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>hsdbvkjsbdkfbkjsd</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_6">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>464</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
